server/views/temporary-accommodation/reports/index.njk

```diff
@@ -2,14 +2,13 @@
 {% extends "../../partials/layout.njk" %}
 {% from "../../partials/showErrorSummary.njk" import showErrorSummary %}
 {% from "../../components/formFields/form-page-select/macro.njk" import formPageSelect %}
-{% from "../../components/datePicker/macro.njk" import hmppsDatepicker %}
+{%- from "moj/components/date-picker/macro.njk" import mojDatePicker -%}
 {% from "govuk/components/back-link/macro.njk" import govukBackLink %}
 
 {% set pageTitle = "Reports - " + applicationName %}
 {% set mainClasses = "app-container govuk-body" %}
 
 {% block extraScripts %}
-    <script type="text/javascript" nonce="{{ cspNonce }}" src="/assets/js/datepicker.js"></script>
```

Comment:
Assume this comes packaged with the MOJ date picker?

Reply:
Thats correct